platforms/XboxOne/builder.js

```diff
@@ -0,0 +1,37 @@
+const utils = require('../../tasks/utils');
+const path = require("path");
+
+module.exports = {
+    build: build
+};
+
+const pkg = utils.readJSON(path.resolve(__dirname, "../../package.json"));
+
+function build(env, serverUrl) {
+    if (!env) env = 'dev';
+    if (!serverUrl) serverUrl = utils.appUrl;
+
+    const destPkgName = `${pkg.name}_xboxone_${env}_${pkg.version}.pkg`;
+    console.log(`building ${destPkgName}`);
```

Comment:
this is not actually used currently.
could you do a final move of the build resuilt to the dist folder with the above name